recipes/r-seurat/meta.yaml

```diff
@@ -0,0 +1,120 @@
+{% set version = '2.2.0' %}
+
+package:
+  name: r-seurat
+  version: {{ version }}
+
+source:
+  fn: Seurat_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/Seurat_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/Seurat/Seurat_{{ version }}.tar.gz
+  sha256: bc4e0fd07b944a980cae70ac26387793be763f0807b973f926d34dea17fc59f9
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-fnn
+    - r-hmisc
+    - r-mass
+    - r-matrix
+    - r-nmf
+    - r-rcolorbrewer
+    - r-rocr
+    - r-rcpp
+    - r-rcppeigen
+    - r-rcppprogress
+    - r-rtsne
+    - r-sdmtools
+    - r-vgam
+    - r-ape
+    - r-caret
+    - r-cowplot
+    - r-diffusionmap
+    - r-dplyr
+    - r-dtw
+    - r-fpc
+    - r-gdata
+    - r-ggplot2
+    - r-ggridges
+    - r-gplots
+    - r-gridextra
+    - r-ica
+    - r-igraph
+    - r-irlba
+    - r-lars
+    - r-mixtools
+    - r-pbapply
+    - r-plotly
+    - r-ranger
+    - r-reshape2
+    - r-stringr
+    - r-tclust
+    - r-tidyr
+    - r-tsne
+    - gcc
+
+  run:
+    - r-base
+    - r-fnn
+    - r-hmisc
+    - r-mass
+    - r-matrix
+    - r-nmf
+    - r-rcolorbrewer
+    - r-rocr
+    - r-rcpp
+    - r-rcppeigen
+    - r-rcppprogress
+    - r-rtsne
+    - r-sdmtools
+    - r-vgam
+    - r-ape
+    - r-caret
+    - r-cowplot
+    - r-diffusionmap
+    - r-dplyr
+    - r-dtw
+    - r-fpc
+    - r-gdata
+    - r-ggplot2
+    - r-ggridges
+    - r-gplots
+    - r-gridextra
+    - r-ica
+    - r-igraph
+    - r-irlba
+    - r-lars
+    - r-mixtools
+    - r-pbapply
+    - r-plotly
+    - r-ranger
+    - r-reshape2
+    - r-stringr
+    - r-tclust
+    - r-tidyr
+    - r-tsne
+    - libgcc
+    - openjdk >=6
+
+test:
+  commands:
+    - $R -e "library('Seurat')"
+
+about:
+  home: http://www.satijalab.org/seurat, https://github.com/satijalab/seurat
```

Comment:
`home` should be a valid URL. Please use
```yaml
  home: http://www.satijalab.org/seurat
  dev_url: https://github.com/satijalab/seurat
```

Reply:
good call !